docs/content/en/functions/NumFmt.md

```diff
@@ -2,7 +2,6 @@
 title: lang.NumFmt
 description: "Formats a number with a given precision using the requested `negative`, `decimal`, and `grouping` options. The `options` parameter is a string consisting of `<negative> <decimal> <grouping>`."
 godocref: ""
-workson: []
```

Comment:
Are you sure this fails with strict unmarshalling? That could be a breaking change. I suspect some may have constructs ala:

```
{{ range .Params.workson }}
{{ end }}
```

Reply:
It failed when there were duplicate keys, which I removed fron the docs site and this repo.

Reply:
Ah, OK, so it's not the empty slice that triggers the error.